### PR TITLE
Update docker-compose and caddy for vaultwarden v1.29.0

### PR DESCRIPTION
**Pull request by toto-xoxo**
New vaultwarden version published today no longer required specific websocket attribute. It uses port 80 as websocket.

### CODE REVIEW

**Comment by toto-xoxo** [comment]
also I realized that you were using an old vaultwarden caddy file example I propose to updating it to the latest version

File: caddy/Caddyfile [hunk 1]
```diff
@@ -1,41 +1,24 @@
...
-
-  # Proxy the Root directory to Rocket
-  reverse_proxy bitwarden:80
+  # This setting may have compatibility issues with some browsers
+  # (e.g., attachment downloading on Firefox). Try disabling this
+  # if you encounter issues.
+  encode gzip
 
+  # Proxy everything Rocket
+  reverse_proxy vaultwarden:80 {
```
Comment (asardaes) [inline caddy/Caddyfile:19]: Did you rename the container in your setup? In the example from this repository, it's still `bitwarden`.

Comment (toto-xoxo) [reply caddy/Caddyfile]: You are true. I've updated it.

Also added support for push notifications introduced in v1.29.0

**Comment by dadatuputi** [comment]
Looks like the head was deleted - I'll let you resubmit @toto-xoxo if you want, otherwise I'll just roll in the changes with my own PR.

**Pull request closed without merging**